src/functions/template-tags/organizer.php

```diff
@@ -541,4 +543,85 @@ function tribe_get_organizers( $only_with_upcoming = false, $posts_per_page = -
 		return $query->have_posts() ? $query->posts : array();
 	}
 
+	/**
+	 * Fetches and returns a decorated post object representing a Organizer.
+	 *
+	 * @since TBD
+	 *
+	 * @param null|int|WP_Post $organizer  The organizer ID or post object or `null` to use the global one.
+	 * @param string|null      $output The required return type. One of `OBJECT`, `ARRAY_A`, or `ARRAY_N`, which
+	 *                                 correspond to a WP_Post object, an associative array, or a numeric array,
+	 *                                 respectively. Defaults to `OBJECT`.
+	 * @param string           $filter Type of filter to apply. Accepts 'raw'.
+	 * @param bool             $force  Whether to force a re-fetch ignoring cached results or not.
+	 *
+	 * @return array|mixed|void|WP_Post|null {
+	 *                              The Organizer post object or array, `null` if not found.
+	 *
+	 *                              @type string $phone The organizer phone number NOT filtered, apply anti-spambot filters if required.
+	 *                              @type string $website The organizer full website URL.
+	 *                              @type string $email The organizer email address NOT filtered, apply anti-spambot filters if required.
+	 *                          }
+	 */
+	function tribe_get_organizer_object( $organizer = null, $output = OBJECT, $filter = 'raw', $force = false ) {
+		/**
+		 * Filters the organizer result before any logic applies.
+		 *
+		 * Returning a non `null` value here will short-circuit the function and return the value.
+		 * Note: this value will not be cached and the caching of this value is a duty left to the filtering function.
+		 *
+		 * @since TBD
+		 *
+		 * @param mixed       $return      The organizer object to return.
+		 * @param mixed       $organizer       The organizer object to fetch.
+		 * @param string|null $output      The required return type. One of OBJECT, ARRAY_A, or ARRAY_N, which
+		 *                                 correspond to a `WP_Post` object, an associative array, or a numeric array,
+		 *                                 respectively. Defaults to `OBJECT`.
+		 * @param string      $filter      Type of filter to apply. Accepts 'raw'.
+		 */
+		$return = apply_filters( 'tribe_get_organizer_object_before', null, $organizer, $output, $filter );
+
+		if ( null !== $return ) {
+			return $return;
+		}
+
+		$post = false;
+		if ( ! $force ) {
+			$cache_key = 'tribe_get_organizer_object_' . md5( json_encode( [ $organizer, $output, $filter ] ) );
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: af79d9155ee022613e435a8d421b5fa8</sub>